CMSgov/design-system
# [WNMGDS-1817, WNMGDS-1818, WNMGDS-1861] remove old doc site content, update a11y tests

## Summary

- Removed old doc site content
- Removed old doc site yarn tasks
- Removed old doc site build functionality from `design-system-scripts`
- Updated `design-system-scripts` and our a11y tests extensively, now uses `storybook-static` builds to test against
- Changed `build:gatsby` to `build:docs`
- Changed `serve:gatsby` to `serve:docs`
- Changed `start` to start gatsby in dev mode
- Removed existing a11y tests, special functionality can be worked in again, in the future

## How to test

Make sure all our build and testing tasks work!

`yarn install && yarn build && yarn build:healthcare && yarn build:medicare`
`yarn build-storybook:gatsby && yarn build:gatsby && yarn serve:gatsby`
`yarn start`
`yarn storybook`
`yarn test:unit`
`yarn test:a11y core` -- note there are violations but all tests show as passing, this is to be fixed in a future ticket
`yarn test:a11y (healthcare/medicare)`
